api/db/migrations/20240325104111_add-doesLogout-and-doesCustomLogout-to-table-oidc-providers.js

```diff
@@ -0,0 +1,19 @@
+const TABLE_NAME = 'oidc-providers';
+const COLUMN1_NAME = 'doesLogout';
+const COLUMN2_NAME = 'doesCustomLogout';
```

Comment:
```suggestion
const COLUMN_NAME = 'shouldEndSession';
```

Je pense qu'une seule variable est nécessaire et peut être trouver un autre nom comme par exemple `shouldEndSession` qui porte sur le faite que l'on doit ou non clôturer la session du partenaire.

On peut faire une mob review avec @1024pix/team-acces pour en discuter.

Reply:
Il a été décidé en équipe de choisir le nom `shouldCloseSession` qui permet d'indiquer qu'il s'agit d'une propriété générique non-proche du mot `endSession` utilisé pour la fonctionnalité de logout standard d'OIDC.